driver/bpf/fillers.h

```diff
@@ -6252,7 +6298,8 @@ FILLER(sched_prog_exec_4, false)
 
 	/* `exe_writable` flag logic */
 	bool exe_writable = false;
-	exe_writable = get_exe_writable(task);
+	struct inode *inode = get_exe_inode(task);
```

Comment:
In `execve_family_flags` filler we actually check that `inode != NULL` before using it. Shall we do the same here?

Reply:
Moreover i think you miss the `exe_upper_layer` in the `sched_prog_exec_4` filler (that is the new filler for exec for arm64 and s390x).